src/components/Hero.js

```diff
@@ -38,22 +38,22 @@ function Hero({handleLogIn}) {
             </li>
             <li>
               <a rel="noreferrer" target="_blank" href="https://github.com/graphql/">
-                <img alt="graphql" src={graphql} />
+                <img alt="graphql" src={graphql} width={141} />
               </a>
             </li>
             <li>
               <a rel="noreferrer" target="_blank" href="https://github.com/vercel/next.js/">
-                <img alt="nextjs" src={next} />
+                <img alt="nextjs" src={next}  width={66} />
               </a>
             </li>
             <li>
               <a rel="noreferrer" target="_blank" href="https://github.com/electron/">
-                <img alt="electron" src={electron} />
+                <img alt="electron" src={electron}  width={261} />
```

Comment:
This would be more flexible if this lived in the css as max-width. The other images could also use the same max-width.